modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/services/rest/TestraySubtask.ts

```diff
@@ -327,6 +328,7 @@ class TestraySubtaskImpl extends Rest<SubtaskForm, TestraySubTask> {
 		}
 
 		const updatedSubtask = await this.update(subTaskId, {
+			dueStatus: currentSubtask?.dueStatus.key,
```

Comment:
Could you check the need for "?", in currentSubtask if dueStatus is a required field.


```suggestion
			dueStatus: currentSubtask.dueStatus.key,
```